iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -521,6 +521,20 @@ private async Task DisposeMessageAsync(string lockToken, AmqpIotDisposeActions o
 
         #endregion Accept-Dispose
 
+        #region Convention-based operations
+
+        public override Task<ClientProperties> GetPropertiesAsync(PayloadConvention payloadConvention, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException("Property operations are currently not supported over AMQP. Please use MQTT protocol instead.");
```

Comment:
the new APIs aren't supported over amqp yet

Reply:
I'm a little worried that users may misconstrue "Property operations" to include twin operations. Since we do support the legacy twin operations over AMQP, is there a better way to phrase this error?

Reply:
Good point, how about:
"This API is currently not supported over AMQP, please use MQTT instead. Note that you can still retrieve the device twin properties using DeviceClient.GetTwinAsync()/ ModuleClient.GetTwinAsync()"

Reply:
updated to:
```csharp
throw new NotImplementedException("This operation is currently not supported over AMQP, please use MQTT protocol instead. " +
    "Note that you can still retrieve a client's properties using the legacy DeviceClient.GetTwinAsync(CancellationToken cancellationToken) or " +
    "ModuleClient.GetTwinAsync(CancellationToken cancellationToken) operations, but the properties will not be formatted " +
    "as per DTDL terminology.");
```